docs/source/README.md

```diff
@@ -0,0 +1,30 @@
+<img src="assets/logo.png" height="320" width="620" >
+
+> A component library for the Guardian's [Source Design System](https://theguardian.design).
+
+[![Storybook](https://cdn.jsdelivr.net/gh/storybookjs/brand@main/badge/badge-storybook.svg)](https://guardian.github.io/source)
+
+## Components ⚛️
+
+See [the Source storybook](https://guardian.github.io/source) for examples of available components.
+
+## Docs 📖
+
+- [Getting started](https://guardian.github.io/source/?path=/story/getting-started--page)
```

Comment:
Given these point to the old storybook instances does it make sense at this point to either update them for the new instances or at least create a ticket to do so?

This also comes back to yesterdays discussion about whether we want the stories for eslint plugins still.

Reply:
Good point Joe, we'll take a look at